vignettes/models.Rmd

```diff
@@ -77,17 +80,19 @@ riskratio(formula = death ~ stage + receptor,
 With `approach = "all"`, all model types listed in the tables are fitted. The fitted object, *e.g.*, `fit`, is one of the converged models. A summary of the convergence status of all models is displayed at the beginning of `summary(fit)`:
 
 ```{r allmodels}
-fit_all <- riskratio(formula = death ~ stage + receptor, 
-                     data = breastcancer, 
-                     approach = "all")
+fit_all <- riskratio(
+  formula = death ~ stage + receptor, 
+  data = breastcancer, 
+  approach = "all"
+)
 summary(fit_all)
 ```
 
 \
 Individual models can be accessed as `fit$all_models[[1]]` through `fit$all_models[[6]]` (or `[[7]]` if fitting a risk ratio model). `tidy()` shows coefficients and confidence intervals from all models that converged:
 
 ```{r allmodels2}
-tidy(fit_all) %>%
-  select(-statistic, -p.value) %>%
+tidy(fit_all) |> 
```

Comment:
I am not sure about introducing a dependency on R 4.1. I still see scientists use 4.0 and earlier.